test/e2e/integration/payment-test/payment.ts

```diff
@@ -0,0 +1,105 @@
+import 'cypress-iframe';
+
+import { faker } from '@faker-js/faker';
+import path from 'path';
+
+import { AppFrontend } from 'test/e2e/pageobjects/app-frontend';
+const appFrontend = new AppFrontend();
+
+let paymentUrl: string;
+
+describe('Payment', () => {
+  beforeEach(() => {
+    cy.startAppInstance(appFrontend.apps.paymentTest, { authenticationLevel: '2' });
+  });
+
+  it('Should fill out the form, landing on the payment page', () => {
+    cy.intercept({
+      method: 'GET',
+      url: '**/ttd/payment-test/*',
+      query: {
+        paymentid: '*',
+      },
+    }).as('paymentRequest');
+    cy.findByRole('radio', { name: /Jeg fyller ut skjemaet som søker/ }).click();
+    cy.contains('button', 'Varer og tjenester').click();
+    cy.findByRole('checkbox', { name: /Jeg har lest og forstått/ }).click();
+    cy.findByRole('textbox', { name: /Klassenummer/ }).type('1');
+    cy.findByRole('textbox', { name: /Varer\/tjenester/ }).type('test');
+    cy.contains('button', 'Betalning og saksgangen vidare').click();
+    cy.findByRole('button', { name: /Til betaling/ }).click();
+    cy.intercept({
+      method: 'GET',
+      url: '**/ttd/payment-test/instances/**/**/payment',
+    }).as('paymentInfoRequest');
+
+    cy.wait('@paymentInfoRequest').then((_) => {
+      cy.url().then((url) => {
+        cy.log(`Current URL is: ${url}`);
+        paymentUrl = url;
+      });
+    });
```

Comment:
So this test is storing a URL in a local variable that is then used by the next test? This would break if using `it.only()`. I think if the tests depend on each other, they're not really testing two different things, so you could just keep them in the same `it()`.